### PR TITLE
Hub 01 laser turrets.

### LINKED ISSUE #41857: Robofac members getting killed by their own security turrets.
# Describe the bug

The CROWS turrets in Hub 01 underground layout aren't, actually, the same faction as Hub 01. As such they will regularly murder the soldiers they are meant to protect.

# Steps To Reproduce

Start a new world, teleport to Hub 01 and enter it by the subway route. There is a security room with four turrets between the subway and base - Just stay in the other side of the security room making noise. The guards will try to investigate, enter the turret room and will quickly be torn to shreds. 

(There are enough guards that they will eventually win and destroy the turrets, but that's hardly good).

# Expected behavior

Hub 01 turrets should be able to recognize Hub 01 members.

Either change the CROWS turrets for a new type that it's specifically part of Hub 01 faction, or just remove them.

# Screenshots

# Versions and configuration

Tested in cataclysmdda-0.E-Windows_x64-Tiles-b10795, no mods.

# Additional context

I was very confused the first time i assaulted the Hub underground, and i wasn't entirely sure if their robots rebelling and murdering them was intended or not, but after looking at the layout it's clearly not. It's just silly.

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary

SUMMARY: Content "Adds laser turrets to hub 01"

#### Purpose of change

This will fix #41857. Figured I needed to add a new monster due to the faction, so I went ahead and added some presumably homebrew laser turrets inside the place. 

#### Describe the solution

Adds a robofac monster faction, npcs from Hub 01 are considered part of that faction.

#### Additional context

Unfortunately, while the turrets consider the NPCs to be friends and wont attack them; the NPC guards dont think the same thing, and will kill the friendly turrets if they spot them for some reason.

As such I closed off the rooms containing the turrets as an aditional measure.

### CODE REVIEW

**Comment by Broken27** [comment]
"Unfortunately, while the turrets consider the NPCs to be friends and wont attack them; the NPC guards dont think the same thing, and will kill the friendly turrets if they spot them for some reason."

Weird.

Oh well.

It should be mostly fine now.

File: data/json/monsters/robofac_robots.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "id": "mon_robofac_laserturret_mk1",
+    "type": "MONSTER",
+    "name": { "str": "prototype laser turret" },
+    "description": "This appears to be an very experimental automated tower.  Plating-less and seemingly half-built, its little more than an over sized laser emitter and a camera, both welded to a swiveling platform.",
```
Comment (Night-Pryanik) [inline data/json/monsters/robofac_robots.json:6]:
```suggestion
    "description": "This appears to be a very experimental automated tower.  Plating-less and seemingly half-built, it's little more than an oversized laser emitter and a camera, both welded to a swiveling platform.",
```

File: data/json/items/generic.json [hunk 2]
```diff
@@ -3025,6 +3025,15 @@
     "to_hit": -3,
     "flags": [ "TRADER_AVOID", "NO_REPAIR" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "broken_robofac_laserturret_mk1",
+    "symbol": ",",
+    "color": "green",
+    "name": { "str": "broken laser turret" },
+    "weight": "110 kg",
```
Comment (Night-Pryanik) [inline data/json/items/generic.json:3034]: Intact turret weights 40 kg, while a broken one weights 110 kg?

**Comment by I-am-Erk** [comment]
Should probably bugreport that faction thing

**Comment by John-Candlebury** [comment]
Agreed!

**Pull request merged**